### PR TITLE
Move CONTRIBUTING.md to hosting.md and build on live site

### PR DESCRIPTION
**Pull request by jnewbery**
... and some minor comments to hosting.md.

### CODE REVIEW

File: hosting.md [hunk 1]
```diff
@@ -14,17 +17,20 @@ Some tips for hosting a great review club meeting:
...
 - In the notes and meeting, don't just talk about the mechanical code changes.
-  Other important aspects to consider are: the historical context of the PR (why
-  things in the code are the way they are currently, and what PRs have touched
-  that part of the code base), testing methodology and motivation for the PR.
+  Other important aspects to consider are:
+
+  - motivation for the PR
+  - the historical context of the PR (why things in the code are the way they
+    are currently, and what PRs have touched that part of the code base in the past)
+  - how to review the changes and testing methodology.
```
Comment (glozow) [inline hosting.md:28]: suggestions (feel free to take none/some/all)

- advantages and disadvantages of this approach
- alternative approaches
- potential bugs reviewers might want to look out for
- fun C++ things

File: hosting.md [hunk 2]
```diff
@@ -1,6 +1,9 @@
-# Hosting a Bitcoin Core Review Club Meeting
+---
+layout: default
+title: Hosting a Review Club Meeting
+---
 
-Some tips for hosting a great review club meeting:
+# Hosting a Bitcoin Core Review Club Meeting
```
Comment (glozow) [inline hosting.md:7]: For here or the future, a section on "how to choose a suitable PR" perhaps? And a note that it's totally fine to host a PR authored by yourself (one person asked me thinking that's not okay)

**Review by glozow — COMMENTED** [review]
very nice

File: hosting.md (hunk 2 above)
Comment (jnewbery) [reply hosting.md:7]: I've had a first attempt at this. Let me know what you think.

**Comment by jnewbery** [comment]
Thanks for the review @glozow. I've updated the hosting.md doc following your suggestions. Let me know what you think.

**Comment by glozow** [comment]
Looks good. Maybe also add hosting "tab" to the menu in _data/settings.yml?

**Comment by jnewbery** [comment]
> Maybe also add hosting "tab" to the menu in _data/settings.yml?

Done!

**Pull request merged**